src/Apps/Artist/Routes/Overview/Components/RecommendedArtist.tsx

```diff
@@ -118,11 +118,33 @@ const RecommendedArtist: FC<
             />
           )
         }}
+        renderLeftArrow={({ Arrow }) => {
+          return (
+            <ArrowContainer>
+              <Arrow />
+            </ArrowContainer>
+          )
+        }}
+        renderRightArrow={({ Arrow }) => {
+          return (
+            <ArrowContainer>
+              <Arrow />
+            </ArrowContainer>
+          )
+        }}
       />
     </>
   )
 }
 
+const ArrowContainer = styled(Box)`
+  align-self: flex-start;
+
+  ${ArrowButton} {
+    height: 60%;
+  }
+`
+
```

Comment:
@jiahaodavid changes to the RecommendedArtist component wasn't a part of this ticket. I wonder if you meant to make this change in the [`ArtistCollectionsRail`](https://github.com/artsy/reaction/blob/35998131eb2c0af1040de7af49095faec0e4b309/src/Apps/Artist/Components/ArtistCollectionsRail/ArtistCollectionsRail.tsx#L92-L98)?

Reply:
Yes. I found out the arrows are not being centered on the artist page. It's same issues like we have on the collection page. That's why I change the artist page file.
<img width="457" alt="Screen Shot 2019-07-17 at 11 33 57" src="https://user-images.githubusercontent.com/45926410/61389612-384afa80-a887-11e9-89af-d022b8b935df.png">

Reply:
Ah gotcha 👌